code/index.html

```diff
@@ -3,28 +3,111 @@
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <title>PASTA WARS</title>
+    <link rel="stylesheet" href="./style.css">
+    <!--here is a link to get an icon on the tab!-->
+    <link rel="icon" href="https://res.cloudinary.com/occsn/image/upload/v1579300893/products/Pasta_Wars_event_pnjw9n.jpg" type="image/png" sizes="16x16">
+    <!--here is the linksheet to get the plus and minus signs for the FAQ-->
+    <link rel="stylesheet" href="https://code.ionicframework.com/ionicons/2.0.1/css/ionicons.min.css">
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
+    <!-- here is the link where i created my logo with: https://codetheweb.blog/fullscreen-image-hero/ -->
+<header>
+     <nav class="navigation">
+      <a class="navigation-link" href="#">Pasta</a>
+      <a class="navigation-link" href="#">Pizza</a>
+      <img class="logo" src="./LOGO.png" alt="Pasta Wars logo with a fork and knife above the words"/>
+      <a class="navigation-link" href="#">Mamma Mia</a>
+      <a class="navigation-link" href="#">Dolmio</a>
+     </nav>
+ </header>
 
-    <!-- Signup form -->
+   <section class="hero">
+   <div class="hero-inner">
+     <H1>PASTA WARS</H1>
+     <H2>DO YOU HAVE WHAT IT TAKES?</H2>
+     <video autoplay muted loop preload="auto" id="myVideo">
+      <source src="./videos/herovideo.mp4" type="video/mp4">
+    </video>
+   </div>
+  </section>
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+      <!-- Signup form -->
+  <Section class="signupForm">
+  <form action="http://httpbin.org/anything" method="POST">
+    <label>
+    Name
+    <input type="text" name="message" required>
+    </label>
+    <label>
+        E-mail
+     <input type="email" name="email" required>
+    </label>
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
+    <label>  
+    Please select a country
+    <select name="country">
+        <option value="SWE">Sweden</option>
+        <option value="JAP">Japan</option>
+        <option value="MON">Mongolia</option>
+    </select>
+    </label>
+    
+    <label class="cheese"> 
+      Dried Pasta:
+      <input type="radio" name="cheese"/>
+     </label>
+    
+     <label class="cheese"> 
+      Fresh Pasta:
+      <input type="radio" name="cheese"/>
+     </label>
+
+    <label>
+      <input type="checkbox" checked="checked" name="remember" style="margin-bottom:15px"> I'd like pasta news
+    </label>
+    <button type="submit">Get your free pasta pot!</button>
+</form>
+</Section>
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+ 
+    <div class="container">
+      <h2>Frequently Asked Questions</h2>
+      <div class="accordion">
+        <div class="accordion-item">
+          <a>What the frick is Pasta Wars?</a>
```

Comment:
Why use a-tag when it's not a link? Am I missing something?

Reply:
Thank you for your input. Im not really sure too since i followed a tutorial on how to build an accordion but here is a good stack oveerflow anwsering how the <a> tag can be used: https://stackoverflow.com/questions/10510191/valid-to-use-a-anchor-tag-without-href-attribute